lib/src/repo.rs

```diff
@@ -617,6 +617,15 @@ impl MutableRepo {
         (self.index, self.view.into_inner())
     }
 
+    pub fn new_commit(
+        &mut self,
```

Comment:
Maybe better to add `#[must_use]` here (or to `struct CommitBuilder`.)

Reply:
Good idea. I added it to the struct.